tests/unit/providers/utils/test_model_registry.py

```diff
@@ -162,26 +162,29 @@ async def test_register_model_existing_different(
         await helper.register_model(known_model)
 
 
-async def test_unregister_model(helper: ModelRegistryHelper, known_model: Model) -> None:
-    await helper.register_model(known_model)  # duplicate entry
-    assert helper.get_provider_model_id(known_model.model_id) == known_model.provider_model_id
-    await helper.unregister_model(known_model.model_id)
-    assert helper.get_provider_model_id(known_model.model_id) is None
+# TODO: unregister_model functionality was removed/disabled by https://github.com/meta-llama/llama-stack/pull/2916
+# async def test_unregister_model(helper: ModelRegistryHelper, known_model: Model) -> None:
+#     await helper.register_model(known_model)  # duplicate entry
+#     assert helper.get_provider_model_id(known_model.model_id) == known_model.provider_model_id
+#     await helper.unregister_model(known_model.model_id)
+#     assert helper.get_provider_model_id(known_model.model_id) is None
 
 
-async def test_unregister_unknown_model(helper: ModelRegistryHelper, unknown_model: Model) -> None:
-    with pytest.raises(ValueError):
-        await helper.unregister_model(unknown_model.model_id)
+# TODO: unregister_model functionality was removed/disabled by https://github.com/meta-llama/llama-stack/pull/2916
+# async def test_unregister_unknown_model(helper: ModelRegistryHelper, unknown_model: Model) -> None:
+#     with pytest.raises(ValueError):
+#         await helper.unregister_model(unknown_model.model_id)
```

Comment:
Same question here

Reply:
Keeping these tests disabled for now is good, I will take care of re-enabling them once I refactor a bit more of the registry logic